frappe/public/js/frappe/form/control.js

```diff
@@ -1379,50 +1379,54 @@ frappe.ui.form.ControlLink = frappe.ui.form.ControlData.extend({
 				me.awesomplete.list = me.$input.cache[doctype][term];
```

Comment:
On input event: Stuff upto here (immediate showing) can be done without a timeout.